css/style.css

```diff
@@ -3,8 +3,16 @@
 
 body {
     font-family: 'Roboto', sans-serif;
+    font-size: 16px;
     -webkit-font-smoothing: antialiased;
-}
+    width:100%;
+    height:100%;
+    color:#838994;
+    text-align: center;
+    line-height: 25px;
+    box-sizing: border-box; /*padding included in the box size*/
+    display: block; /*each element as a block*/
```

Comment:
```suggestion
```
Normally, it's not a good idea to turn every element into a block, because when we use inline elements like `<a>` and `<strong>` for example, they will all be on separate lines which does not make sense